app/common/renderer/actions/Session.js

```diff
@@ -492,6 +492,19 @@ export function newSession(caps, attachSessId = null) {
         desiredCapabilities['mobitru:options'].source = 'appium-inspector';
         break;
       }
+      case SERVER_TYPES.TVLABS: {
+        host = process.env.TVLABS_WEBDRIVER_URL || 'appium.tvlabs.ai';
+        path = '/';
+        port = 4723;
+        https = host === 'appium.tvlabs.ai' ? true : false;
```

Comment:
`host === 'appium.tvlabs.ai' ? true : false` is redundant. `https = host === 'appium.tvlabs.ai'` should be enough

Reply:
```suggestion
        https = host === 'appium.tvlabs.ai';
```